modules/passive-circuits.stanza

```diff
@@ -48,7 +48,7 @@ public pcb-module voltage-divider ( input-voltage:Double
   pin in
   pin out
   pin lo
-  val solution = voltage-divider-solver(tol%(input-voltage, 0.0), tol%(output-voltage, 10.0), current, tolerance / 100.0, 0.15)
+  val solution = voltage-divider-solver(typ(input-voltage), typ(output-voltage), current, tolerance / 100.0, 0.15)
```

Comment:
Why remove the tolerance target on the output voltage? Does the +/- 10% not useful?

Reply:
That's a mistake.  reverting that part